### ISSUE TITLE
Columns don't populate successfully if data length > 8192

### ISSUE THREAD

**Issue by davewthompson** [ratanakvlun/sequelize-msnodesqlv8#6]
If a column contains data longer than 8192 characters then the resulting object will only contain the last 'packet' of data. This is because the underlying _msnodesqlv8_ will invoke the 'column' event multiple times  with the same column ID but different data, and the previous data inside the _rowBuffer_ array is overwritten.

I have fixed this just by doing (request.js:48):

```
let existing = rowBuffer[index];

if(existing && existing.metadata.colName === columnMetadata.name && existing.value) {
    existing.value += data;
    return;
}
```

My apologies for not creating a pull request, but the organisation I work for doesn't allow us to contribute code back to GitHub.

**Comment by ratanakvlun** [comment]
Sorry for the delay. I appreciate you pointing this out. I'll get this fix in ASAP.

**Comment by ratanakvlun** [comment]
Fixed in 0.2.5.
